app/kubemci/pkg/gcp/urlmap/urlmapsyncer.go

```diff
@@ -70,14 +72,21 @@ func (s *URLMapSyncer) EnsureURLMap(lbName string, ing *v1beta1.Ingress, beMap b
 	if err == nil {
 		fmt.Println("url map", name, "exists already. Checking if it matches our desired url map", name)
 		glog.V(5).Infof("Existing url map: %v\n, desired url map: %v", existingUM, desiredUM)
+		// TODO(G-Harmon): Figure out how to properly calculate the fp. Using Sha256 returned a googleapi error.
```

Comment:
I think reusing the existing fp is fine rather than trying to recompute it?
I see thats what we do in ingress-gce: https://github.com/kubernetes/kubernetes/blob/9af339885b68990a96470cc4350514d2a1ec30c8/pkg/cloudprovider/providers/gce/gce_loadbalancer_internal.go#L540

Reply:
I think you're supposed to change the fingerprint if you've changed the contents. (That comment talks about optimistic locking... I guess maybe the server would avoid work if nothing was changed?)